RecoLocalMuon/RPCRecHit/src/CSCObjectMap.cc

```diff
@@ -9,14 +9,8 @@
 #include "RecoLocalMuon/RPCRecHit/src/CSCObjectMap.h"
 #include "RecoLocalMuon/RPCRecHit/src/CSCStationIndex.h"
 
-CSCObjectMap::CSCObjectMap(MuonGeometryRecord const& record) {
-  edm::ESHandle<RPCGeometry> rpcGeo;
-  record.get(rpcGeo);
-
-  edm::ESHandle<CSCGeometry> cscGeo;
```

Comment:
If you remove this (useless, indeed) you can also remove the include of "Geometry/CSCGeometry/interface/CSCGeometry.h"

Reply:
done